### PR TITLE
Removed default menus, added new font to menu, got rid of main menu u…

### PR DESCRIPTION
**Pull request by ConnorSheremeta**
…nneeded content, removed saves

### CODE REVIEW

File: HITDAD/js/rpg_windows.js [hunk 1]
```diff
@@ -1812,7 +1805,7 @@ Window_MenuStatus.prototype.setPendingIndex = function(index) {
 // The window for selecting a target actor on the item and skill screens.
 
 function Window_MenuActor() {
-    this.initialize.apply(this, arguments);
+    fthis.initialize.apply(this, arguments);
```
Comment (vppatel111) [inline HITDAD/js/rpg_windows.js]: Was this intentional or is this a typo?

**Comment by vppatel111** [comment]
A couple more issues:

- It appears that the pressing the "Esc" key now does nothing. This is a problem because you can't back out of "Continue" menu or pause during a game.
- Additionally, you removed the "Options" option from the menu but we still want to allow the player to change the game volume from the title screen, we only wanted to remove the TMJumpAction configuration, please see here: https://sevenpointsoffailure.slack.com/archives/CSDFVG2UA/p1584633475047700

Note: You can remove menu options from the ingame menu from Database > System > Menu Commands. You only need to change the code for TMJumpAction to remove their menu item, the others can be done through the interface.

**Review by vppatel111 — CHANGES_REQUESTED** [review]
See above comments.

**Comment by ConnorSheremeta** [comment]
Menus now removed properly, removed typo, esc now works, all options now on options selection again other than jap option (we want all others right?), save and continue disabled

**Review by mikegreber — APPROVED** [review]
Looks good, i merged my map-updates in here and did a fix on a bug that showed up with the level names not displaying properly. For some reason the menu font doesn't load in when the game is initially loaded but we can figure that out later.

**Pull request merged**